modules/fileout_covjson/FoDapCovJsonTransform.h

```diff
@@ -602,6 +610,7 @@ class FoDapCovJsonTransform: public BESObj {
      * @throw BESInternalError if the DDS* is null or if localfile is empty.
      */
     FoDapCovJsonTransform(libdap::DDS *dds);
+    FoDapCovJsonTransform(libdap::DMR *dmr);
```

Comment:
This constructor, and the one above, should be marked with the 'explicit' keyword. That will prevent the compiler from using these in type casts.

Reply:
I will add the explicit to these two methds.